fitlins/interfaces/bids.py

```diff
@@ -157,14 +157,14 @@ def _run_interface(self, runtime):
         selectors = self.inputs.selectors
 
         analysis = ba.Analysis(model=self.inputs.model, layout=layout)
-        analysis.setup(**selectors)
+        analysis.setup(drop_na=False, derivatives='only', **selectors)
```

Comment:
I think this is saying that it won't use event files in the original BIDS directory? That's going to break probably everything but Neuroscout-based analyses.

Reply:
Oh, I meant not to push that. On that note though, since I need to set that to "only" could we make that configurable?

I think this is a good example of a feature that would be useful to allow access to via a Python API, but is probably overkill for the CLI.

Reply:
Sure, how about adding to the input spec:

```Python
derivatives = traits.Enum('both', 'ignore', 'only', desc='...')
```

And adapt [this](https://github.com/INCF/pybids/blob/master/bids/grabbids/bids_layout.py#L225-L228) for the `desc`.